generated/java/proto-google-cloud-pubsub-v1/build.gradle

```diff
@@ -10,7 +10,7 @@ apply plugin: 'signing'
 
 description = 'PROTO library for proto-google-cloud-pubsub-v1'
 group = "com.google.api.grpc"
-version = "0.2.1"
+version = "0.3.0"
```

Comment:
Shouldn't the api-common dependency change below? Did you generate based on your update to googleapis?

Reply:
Ahh I will regenerate everything ok. This PR was just a sed job.